jme3-lwjgl3/src/main/java/com/jme3/system/lwjgl/LwjglWindow.java

```diff
@@ -45,6 +45,11 @@
 import com.jme3.system.JmeSystem;
 import com.jme3.system.NanoTimer;
 import com.jme3.util.BufferUtils;
+import com.jme3.util.SafeArrayList;
+import org.lwjgl.Version;
+import org.lwjgl.glfw.*;
```

Comment:
Please expand this star import to show which classes are imported.

Reply:
Done.